bundles/org.openhab.binding.opengarage/src/main/java/org/openhab/binding/opengarage/internal/OpenGarageConfiguration.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2010-2019 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.opengarage.internal;
+
+/**
+ * The {@link OpenGarageConfiguration} class contains fields mapping thing configuration parameters.
+ *
+ * @author Paul Smedley - Initial contribution
+ */
+public class OpenGarageConfiguration {
+
+    /**
+     * Sample configuration parameter. Replace with your own.
+     */
+    public String host = null;
```

Comment:
remove the comment above (and the empty line). Why do you initialize with `null`?

Reply:
I believe this is addressed in 4714fc9b7d